contracts/Deb0x.sol

```diff
@@ -208,11 +252,13 @@ contract Deb0x is ERC2771Context, ReentrancyGuard {
         gasWrapper(nativeTokenFee)
         gasUsed(feeReceiver, msgFee)
     {
+        require(msgFee < 10001, "Deb0x: Reward fees can not exceed 100%");
+        require(to.length == payload.length, "Deb0x: Arrays must have same length");
+        require(to.length > 0, "Deb0x: Length of arrays must be greater than 0");
```

Comment:
use error messages suggested in [updateToBytes](https://github.com/deb0x/deb0x-faberweb3/pull/84) PR